test/fixtures/basic/nuxt.config.js

```diff
@@ -15,5 +15,10 @@ module.exports = {
     bad: null,
     '': true
   },
-  transition: false
+  transition: false,
+  build: {
+    postcss: [
+      require('postcss-cssnext')()
```

Comment:
Maybe would be better if we use a different plugin like rtlcss for tests. As cssnext is also in default configs so tests will fail even if this option totally being ignored! And resulting into false-positive coverage ;)

Reply:
yes, that's a good idea, will refactor it. 😄

Reply:
Indeed that just was a point so maybe not a priority thing. Thanks mate!

Reply:
ok ok, I will do that when time permits, thanks for the reviewing, bro 😄